sklearn/metrics/ranking.py

```diff
@@ -184,6 +185,84 @@ def _binary_average_precision(y_true, y_score, sample_weight=None):
                                  average, sample_weight=sample_weight)
 
 
+def detection_error_tradeoff(y_true, y_score, pos_label=None,
+                             sample_weight=None):
+    """Compute error rates for different probability thresholds
+
+    Note: this implementation is restricted to the binary classification task.
+
+    Parameters
+    ----------
+    y_true : array, shape = [n_samples]
+        True targets of binary classification in range {-1, 1} or {0, 1}.
+
+    y_score : array, shape = [n_samples]
+        Estimated probabilities or decision function.
+
+    pos_label : int, optional (default=None)
+        The label of the positive class
+
+    sample_weight : array-like of shape = [n_samples], optional
+        Sample weights.
+
+    Returns
+    -------
+    fps : array, shape = [n_thresholds]
```

Comment:
If I am not mistaken the DET curve should return *rates* not counts. This looks like a copy/paste legacy which we should update accordingly.